Arduino/Arduino_config.py

```diff
@@ -0,0 +1,76 @@
+import requests
+
+
+URL = "http://call-flow-testing.appspot.com/"
+
+
+class arduino_Callflow():
+    def get_nodes(self):
+        print(requests.get(URL + "nodes").json())
+
+    def get_node(self, node):
+        print(requests.get(URL + "nodes/{}".format(node)).json())
```

Comment:
if you are using python3 (which you should be using) you can use f-strings to make string-interpolation a bit easier to read.
`"nodes/{}".format(node))` can be rewritten as `f"nodes/{node}"`

Reply:
Also i would use single quotes whenever possible